api/src/main/java/jakarta/servlet/http/LocalStrings.properties

```diff
@@ -19,8 +19,10 @@
 # Default localized string information
 # Localized for Locale en_US
 
-err.cookie_name_is_token=Cookie name \"{0}\" is a reserved token
+err.cookie_name_is_token=Cookie name \"{0}\" contains a reserved token
 err.cookie_name_blank=Cookie name must not be null or empty
```

Comment:
I think this should be:
`Cookie name \"{0}"\" is a reserved token or contains an invalid character for a cookie name`

Reply:
is the property name write? 
Also the translation files are very out of date!

Reply:
Fair point re the property name. Maybe `err.cookie_name_is_invalid`? Same for attribute name below.